google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineApplicationInfoPanel.java

```diff
@@ -47,35 +46,40 @@
  */
 public class AppEngineApplicationInfoPanel extends JPanel {
```

Comment:
Because there were a few issues with this panel. Namely, when refresh() is sent a null ProjectSelectionChangedEvent. It's possible that this could have been handled separately, but I didn't want to go over the overhead of setting up new branches, parallel PRs, etc.